internal/pkg/properties/properties.go

```diff
@@ -6,10 +6,21 @@ import (
 	"os"
 	"sort"
 	"strings"
+
+	"github.com/confluentinc/cli/internal/pkg/utils"
 )
 
-// FileToMap reads key=value pairs from a properties file, ignoring comments and empty lines.
-func FileToMap(filename string) (map[string]string, error) {
+// GetMap reads newline-separated configuration files or comma-separated lists of key=value pairs, and supports configuration values containing commas.
+func GetMap(config []string) (map[string]string, error) {
+	if len(config) == 1 && utils.FileExists(config[0]) {
+		return fileToMap(config[0])
+	}
+
+	return ConfigFlagToMap(config)
```

Comment:
What happens if a user passes in a config file name that doesn't exist, but has a `=` in it (i.e. `config[0] = "temp=config.json"`)? Seems like we'll actually convert this into a `map[string]string` without erroring, but we should throw a file doesn't exist error.

Reply:
Hmm, I would expect a backend error in this case.

Reply:
Yeah, looks like for the current usages of `GetMap` this should be fine (I still think we should error if the file isn't found found so we don't make unnecessary API requests).

If we decide to bring back `config` commands and have the `set` command accept `key=value` pairs as arguments to mirror this syntax, this bug will probably resurface anyways.